### DISCUSSION TITLE
How are different column between Cases Proactive and  Cases Proactve in File cases_by_province.csv ?

### DISCUSSION THREAD

**Discussion by zhuzhuzhutai** [djay/covidthailand#202] [Q&A] [answered]
I see two columns(Cases Proactive and  Cases Proactve ) in File cases_by_province.csv .
I don't know how are they different?

**Comment by djay** [comment, marked as answer]
I think this is a bug. I suspect its coming from two different sources. The daily briefing document and ddc dashboard but the code should be combining it into a single column so I'll raise a bug to fix this
